sdk/python/feast/feature_view.py

```diff
@@ -63,24 +62,23 @@ def __init__(
         name: str,
         entities: List[str],
         ttl: Optional[Union[Duration, timedelta]],
-        input: DataSource,
-        batch_source: Optional[DataSource] = None,
```

Comment:
This is an API breaking change - we probably want to raise deprecation warnings or something for a release before killing it since it will break existing users immediately on upgrading feast. wdyt?